.vscode/settings.json

```diff
@@ -27,4 +27,14 @@
     "python.linting.flake8Enabled": true,
     "autoDocstring.docstringFormat": "google",
     "python.formatting.provider": "black",
+    "workbench.colorCustomizations": {
+        "activityBar.activeBackground": "#ab307e",
+        "activityBar.activeBorder": "#25320e",
+        "activityBar.background": "#ab307e",
+        "activityBar.foreground": "#e7e7e7",
+        "activityBar.inactiveForeground": "#e7e7e799",
+        "activityBarBadge.background": "#25320e",
+        "activityBarBadge.foreground": "#e7e7e7"
+    },
+    "python.pythonPath": "/Users/bishwarup.b/opt/anaconda3/envs/optical2/bin/python",
```

Comment:
Please remove the local python path setting in vscode settings.

Reply:
zzz, this happens everytime. I have no idea why. Maybe cause i am switching between container and local. Will fix.